### PR TITLE
Removing login and challenge parameters

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1693431

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Removing `login` and `challenge` parameters per discussion with @spadgett 

@spadgett @enj PTAL 

cc @huffmanca FYI

**Comment by spadgett** [comment]
lgtm, but would be good to have @enj review

File: modules/identity-provider-oidc-CR.adoc [hunk 1]
```diff
@@ -22,43 +22,35 @@ metadata:
...
         name: idp-secret
-      claims: <7>
+      claims: <5>
         preferredUsername:
         - preferred_username
         name:
         - name
         email:
         - email
       urls:
```
Comment (enj) [inline modules/identity-provider-oidc-CR.adoc]: `urls` was dropped openshift/api#265 and replaced with `issuer`

**Comment by enj** [comment]
LGTM, feel free to update OpenID in a different PR

File: modules/identity-provider-oidc-CR.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/identity-provider-oidc-CR.adoc]: Thanks for catching that! This is now updated.

**Comment by ahardin-rh** [comment]
@stuartchuan Can you please provide QE review?

File: modules/identity-provider-oidc-CR.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/identity-provider-oidc-CR.adoc]: Discussed this with @huffmanca and he will provide further edits in a follow-up PR

**Comment by ahardin-rh** [comment]
@barleyer  Can you please provide QE review?

**Comment by barleyer** [comment]
/lgtm

Need an another PR to update openid `issuer` field.

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**